catalogue/webapp/pages/search/images.tsx

```diff
@@ -63,21 +65,32 @@ const BottomPaginationWrapper = styled(PaginationWrapper)`
 const ImagesSearchPage: NextPageWithLayout<Props> = ({
   images,
   imagesRouteProps,
+  query,
 }): ReactElement<Props> => {
-  const { query, page, color: colorFilter } = imagesRouteProps;
+  const { query: queryString } = query;
 
   const { setLink } = useContext(SearchContext);
   useEffect(() => {
     const link = toLink({ ...imagesRouteProps }, 'images_search_context');
     setLink(link);
   }, [imagesRouteProps]);
 
+  // If there is no query, return an empty page
+  if (!queryString) {
+    return (
+      <Space
```

Comment:
Not a huge fan of the empty component but we want spacing and I'm thinking this is still subject to change? Open to suggestions!